server/controllers/user.controller.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/named
```

Comment:
Why disable this eslint rule @cupidcoder ?

Reply:
I will find a workaround this.